vnp/transform.js

```diff
@@ -0,0 +1,5 @@
+msg.body=msg.body.replace('{NHS_NUMBER}', entry); 
+
```

Comment:
What are the numbers for in this script? Can we use variables so their purpose is clearly understood?

Reply:
The `transform.js` file is executed for every line of the [-W](https://github.com/nhsconnect/integration-adaptor-gp2gp/pull/361/files#diff-b92f39385cdea983f292abb288f9aeb980befaa048abdff895908604eaa6f5a2R48) replacing the `{NHS_NUMBER}` placeholder with the `entry` variable that is filled in by the "A" tool by values from given file.
An example has been provided [here](https://github.com/nhsconnect/integration-adaptor-gp2gp/pull/361/files#diff-48b3e573be33df3198bc213bd07a6cf170b1e25feebe4b7e57074b78a45df696)
And usage is [here](https://github.com/nhsconnect/integration-adaptor-gp2gp/pull/361/files#diff-b92f39385cdea983f292abb288f9aeb980befaa048abdff895908604eaa6f5a2R52)

Reply:
Sorry i may have not been clear, i am talking about the numbers in the UUID generate code

Reply:
those are just template parameters to properly format generated UUID